Backfill PyLadies alumni from git history

Kobzol opened the pull request:
https://github.com/PyLadiesCZ/pyladies.cz/pull/792 recently added an alumni section to the PyLadies website. However, most of the actual alumni are now missing from the web, since they were simply deleted in the past. This PR attempts to rectify that, by backfilling the history of alumni from ~~a time machine~~ git history.  The script can be found [here](https://gist.github.com/Kobzol/217769d9cdf03074fe035047c113994f). It quite possible that it isn't perfect, but it seems like a lot of people were backfilled, so it's a start.

There were some cities with a massive number of alumni, which forced me to also modify the design of the alumni section. I fitted 10 people per row, made the font smaller, and also outright hid it on mobile devices, because the list could be very long (looking at you, *Brno*). I also hid roles and links for alumni, I think that name + photo is enough. I can also remove this data from the YAMl if you want, but it's in the git history of this repo anyway :man_shrugging:

Here is the compressed and zoomed out look of the Brno page now:
![image](https://github.com/user-attachments/assets/38b386d2-e82f-4d3a-ac9e-d01c1eab11a6)

An alternative could be to only display alumni on demand, e.g. after a button click, or make the section expandable.

befeleme commented:
For Brno, I'd like to make a smaller review of people: mainly regarding the preferred/legal names that changed in time.

Kobzol commented:
Sure! I already found a few situations where someone was included twice, most of them were surname changes after a wedding (or at least that's my best guess, hopefully I got the before/after name correctly :) ).

frenzymadness commented on teams/ostrava.yml:
```diff
@@ -54,3 +54,36 @@
+  - linkedin-link: https://www.linkedin.com/in/filip-valder-7296a3122
+  - mail-link: mailto:filip.valder@gmail.com
+
+- name: Štěpán Tomsa
+  alumni: true
+  img: img/ostrava/team/stepan_tomsa.jpg
+  role: Kouč
+  links:
+  - twitter-link: https://twitter.com/Glutexo
+  - mail-link: mailto:stepantomsa@icloud.com
```
Duplicita: Štěpán Tomsa alumni == Glutexo v aktivních členech.

Kobzol replied:
Díky! Hledal jsem to podle duplicitních fotek na disku, ale to tady nevyšlo.

frenzymadness reviewed (approved):
I like the idea as well as its implementation. :+1: from me. I'd ask other cities on Slack and give them some time to respond before merging.

befeleme commented on teams/brno.yml:
```diff
@@ -124,11 +124,707 @@
+  - github-link: https://github.com/tnevrlka
+  - linkedin-link: https://www.linkedin.com/in/tnevrlka
+
+- name: Terezie Vahalová
```
cleared with Terka:

```suggestion
- name: Terezie Viktorinová
```

befeleme commented on teams/brno.yml:
```diff
@@ -124,11 +124,707 @@
+  links:
+  - mail-link: mailto:tomas.n@orsava.cz
+
+- name: Katka Chmelařová
+  alumni: true
+  img: img/brno/team/blank.png
+  role: Kouč
+  links:
+  - mail-link: mailto:sprinarova.katka@gmail.com
```
Duplicite entry: let's remove it

befeleme commented on teams/brno.yml:
```diff
@@ -124,11 +124,707 @@
+  img: img/brno/team/blank.png
+  role: Koučka
+
+- name: Edita Lelkes
+  alumni: true
+  img: img/brno/team/blank.png
+  role: Koučka
+  links:
+  - mail-link: mailto:balkoedi@gmail.com
+
```
Duplicity: let's remove this entry (Edit prefers being Edit ;))

befeleme commented on teams/brno.yml:
```diff
@@ -124,11 +124,707 @@
+  links:
+  - mail-link: mailto:juraj.michalek132@gmail.com
+
+- name: Kateřina Chmelařová
+  alumni: true
+  img: img/brno/team/blank.png
+  links:
+  - mail-link: mailto:sprinarova.katka@gmail.com
```
Duplicite: let's remove this entry (Káťa prefers Káťa which is listed below)

befeleme commented on teams/brno.yml:
```diff
@@ -124,11 +124,707 @@
+  links:
+  - mail-link: mailto:yaplik@gmail.com
+
+- name: Edita Balkóová
+  alumni: true
+  img: img/brno/team/blank.png
+  role: Kouč
+  links:
+  - mail-link: mailto:386966@mail.muni.cz
```
Duplicite, we've got Edit under her current name.

befeleme reviewed (changes requested):
There are some duplicates and changes to be made - first pass.

Kobzol commented:
Thanks! Removed the duplicates and rebased. I also modified my script to find duplicates based on e-mails, rather than just images, and found one more typo - `Hana Štamberková` is currently on the website, but git history contains `Hana Štramberková`.

I think her name is `Štamberková`? But the e-mail is `hana.stramberkova@pyladies.cz` - is there also a typo in the e-mail?

befeleme commented:
> Thanks! Removed the duplicates and rebased. I also modified my script to find duplicates based on e-mails, rather than just images, and found one more typo - `Hana Štamberková` is currently on the website, but git history contains `Hana Štramberková`.
> 
> I think her name is `Štamberková`? But the e-mail is `hana.stramberkova@pyladies.cz` - is there also a typo in the e-mail?

Yeah. Remove the "alumni" entry. The e-mail is, I'm afraid, correct. I was not aware I made a typo when creating the address years ago :facepalm:

mcurlej commented:
Maybe add scrolling when there is more then specific count of people? Or move it lower on the page?

ZelenyMartin commented:
> I think her name is `Štamberková`? But the e-mail is `hana.stramberkova@pyladies.cz` - is there also a typo in the e-mail?

Email address was created with a typo, but Hanka wants to keep it like that.

ZelenyMartin reviewed (approved):
I tested this PR locally and looks good to me.

Kobzol commented:
@frenzymadness Looks like others seem to be fine with this PR. Do you want to wait for more people or can we merge it? :)

frenzymadness commented:
I'd merge it. If nobody stops me, I'll do it tomorrow.
